test/core/renderControllerTests.ts

```diff
@@ -1,8 +1,80 @@
 ///<reference path="../testReference.ts" />
 
 describe("RenderController", () => {
-  // HACKHACK: #2083
-  it.skip("Components whose render() is triggered by another Component's render() will be drawn", () => {
+  let SVG_WIDTH = 400;
+  let SVG_HEIGHT = 300;
+
+  describe("configuring the render policy", () => {
+    let oldWarn: (msg: string) => void;
+    let warned: boolean;
+
+    before(() => {
+      oldWarn = Plottable.Utils.Window.warn;
+      Plottable.Utils.Window.warn = () => {
+        warned = true;
+      };
+    });
+
+    beforeEach(() => {
+      warned = false;
+    });
+
+    after(() => {
+      Plottable.RenderController.renderPolicy(Plottable.RenderController.Policy.IMMEDIATE);
+      Plottable.Utils.Window.warn = oldWarn;
+    });
+
+    it("can set a render policy", () => {
+      let renderPolicy = Plottable.RenderController.Policy.TIMEOUT;
+      Plottable.RenderController.renderPolicy(renderPolicy);
+
+      assert.strictEqual(Object.getPrototypeOf(Plottable.RenderController.renderPolicy()),
+        Plottable.RenderPolicies.Timeout.prototype, "render policy is of the same type");
+    });
+
+    it("throws a warning for unrecognized render policies", () => {
+      let unrecognizedRenderPolicy = "foo";
+      Plottable.RenderController.renderPolicy(unrecognizedRenderPolicy);
+
+      assert.isTrue(warned, "warning sent for unrecognized render policy");
```

Comment:
Which part of the warning message do you think is important?  Whenever it comes to this specific portion, I get a bit nervous since if we decide to change the warning message, tests like this will fail.